.github/workflows/link-checker-intermittent.yml

```diff
@@ -0,0 +1,18 @@
+name: Link Checker
+on:
+  schedule:
+    - cron: '0 0 * * 0'  # every Sunday at midnight
+  workflow_dispatch
```

Comment:
Dont see `Run workflow` buttons yet, expect them to appear after merge.

Reply:
The workflow fails (as in: [it fails to run]()https://github.com/eWaterCycle/ewatercycle.github.io/actions/runs/704990310); not sure how to fix it... given that the manual workflow is less important I removed it for now, but I'm open to suggestions!

Reply:
i think it needs a colon (`:`) after workflow_dispatch. See page_build example on https://docs.github.com/en/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration

Reply:
Would that work even without things to follow? I've implemented it but do not see it show up in the actions tab yet... maybe when merged?